template/package.temp

```diff
@@ -81,9 +81,11 @@
     "yaml-lint": "^[yamlLint]"
   },
   "browserslist": [
-    "last 4 version",
-    "> 1%",
-    "IE 11"
+    "last 2 version",
```

Comment:
See [comment on associated issue](https://github.com/AlaskaAirlines/WC-Generator/issues/41#issuecomment-702415932). I would like to be even more explicit about what we support. `last 2 version` brings in last 2 versions of all browsers and includes stuff like Opera Mini.